internal/engine/eval/jq/jq_test.go

```diff
@@ -271,6 +317,27 @@ func TestValidJQEvals(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "valid single rule evaluates constant bool",
+			assertions: []*pb.RuleType_Definition_Eval_JQComparison{
+				{
+					Constant: &structpb.Value{
+						Kind: &structpb.Value_BoolValue{
+							BoolValue: false,
+						},
```

Comment:
I think you can write this as:

```suggestion
					Constant: &structpb.NewBoolValue(false),
```